src/screens/UserPortal/People/People.tsx

```diff
@@ -1,14 +1,15 @@
 import React, { useEffect, useState } from 'react';
 import PeopleCard from 'components/UserPortal/PeopleCard/PeopleCard';
-import { Dropdown, Form, InputGroup } from 'react-bootstrap';
+import { Dropdown, Form, Button } from 'react-bootstrap';
 import PaginationList from 'components/PaginationList/PaginationList';
 import {
   ORGANIZATIONS_MEMBER_CONNECTION_LIST,
   ORGANIZATION_ADMINS_LIST,
 } from 'GraphQl/Queries/Queries';
 import { useQuery } from '@apollo/client';
-import { FilterAltOutlined, SearchOutlined } from '@mui/icons-material';
-import styles from './People.module.css';
+import SearchOutlinedIcon from '@mui/icons-material/SearchOutlined';
+import { FilterAltOutlined } from '@mui/icons-material';
+import styles from '../../../style/app.module.css';
```

Comment:
1. Make this file use CSS exclusively from `src/style/app.module.css`, with the exception of package based CSS.
1. Delete the CSS file it currently uses.
1. Use no CSS in `assets/css/app.css`
1. This will help with future attempts to having Dark Mode
1. Use a variable from `src/style/app.module.css` .
1. If this is the sidebar color, name it `--side-bar` , or whatever this section of code is defining. 

- https://github.com/PalisadoesFoundation/talawa-admin/issues/3296